extensions/ohif-vtk-extension/package.json

```diff
@@ -39,7 +39,10 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.4.5",
+    "css-loader": "^3.0.0",
```

Comment:
I had to add these three modules to run the project locally.
Is it really necessary to be added here?

Reply:
Those should be devDependencies, not dependencies. You can add them if they are necessary, but I don't know why this error would not have shown up earlier.

Reply:
I've updated my `yarn` and suddenly it worked without these dependencies! :)
I'll remove them.